README.md

```diff
@@ -4,227 +4,178 @@
-
-The project's source code will be available in `~/apisonator` and sync'ed with
-your local apisonator directory, so you can edit files in your preferred
-environment and still be able to run whatever you need inside the Docker
-container.
+This is Red Hat 3scale API Management Platform's Backend.
 
-The listener service port (3000 by default) is automatically forwarded to the host machine. The `make dev` command can be executed with the environment variable `PORT` set to
-a different desired listener port value to forward on the host machine.
+It has the following components:
 
-This Docker container is persistent, so your changes will be kept the next time
-you enter it.
+### Apisonator listener
 
-Getting rid of the persistent container is done with `make dev-clean`, whereas
-removing its image is done using `make dev-clean-image`.
+It provides the point of entry of the API Management Platform's Backend. The Service management API (SM API) is provided to authorize and report consumer
+API requests.
 
-Alternatively you can start a container with the service running with
-`make dev-service`. This rule is intended for when you want to test
-the service endpoint.
+Three main operations can be performed with this API:
 
-#### Maintain your dependencies up-to-date
+ * Report: Reports a number of hits to one or more metrics
+ * Authorize: Authorize a request
+ * Authrep: Combination of both the above
 
-Changes in code sometimes translate into changes in dependencies. If that is the
-case, your container will lag behind in dependencies, and some things might just
-start breaking. You might want to make a habit of making sure dependencies are
-updated when you enter your container. Run this from the project directory:
+Make sure to read the corresponding documentation in [ActiveDocs](https://support.3scale.net/reference/active-docs) for
+more information.
 
-> $ `bundle install`
+It attempts to respond with lowest possible latency, performing minimal work and offloading other work to backend-workers by enqueuing
+tasks into job queues.
 
-The container image has additional tools to handle these dependencies for
-multiple Ruby versions. Check out the `scripts` directory if you are curious.
+This component needs access to a Redis database to perform the following actions:
+ * Enqueue reports, which will be processed by the Apisonator worker component
+ * Perform authorization of the requests
+This two actions can be configured to be performed in different Redis databases if desired (see the Configuration section).
 
-#### Workflow
+Finally, another API is provided to configure services in Apisonator. This API is named
+'Internal API'. You can also generate the documentation of the Internal API with Rake tasks.
 
-Your project directory within the container is sync'ed with your local clone of
-the project so that changes in one reflect instantly in the other.
+For a quick Apisonator test, Random services can be created and configured in backend via
```

Comment:
backend => apisonator

Reply:
perhaps even rephrase it to not have to mention apisonator twice

Reply:
:+1: